lib/task_2/task2.dart

```diff
@@ -0,0 +1,35 @@
+// Реализуйте методы для преобразования целых чисел
+// из десятичной системы в двоичную и обратно.
+
+int convertBinaryToDecimal(int binaryNumber) {
+  int decimalNumber = 0;
+  int base = 1;
+
+  while (binaryNumber > 0) {
+    int lastDigit = binaryNumber % 10;
+    binaryNumber = binaryNumber ~/ 10;
+    decimalNumber += lastDigit * base;
+    base = base * 2;
+  }
+  return decimalNumber;
+}
+
+String? convertDecimalToBinary(int decimalNumber) {
```

Comment:
```suggestion
String convertDecimalToBinary(int decimalNumber) {
```
Результат здесь никогда не будет null, поэтому знак вопроса убираем.